server-spi-private/src/main/java/org/keycloak/protocol/ProtocolMapper.java

```diff
@@ -31,8 +31,6 @@
  */
 public interface ProtocolMapper extends Provider, ProviderFactory<ProtocolMapper>,ConfiguredProvider {
     String getProtocol();
-    String getDisplayCategory();
```

Comment:
There are other places where both of these methods are defined whereas at the same time, the `ConfiguredProvider` interface is a parent interface.

Both methods should be part of `ConfiguredProvider` as they are related to metadata associated with a component so that it can be properly rendered in forms.